blocks/library/image/block.js

```diff
@@ -295,4 +306,12 @@ export default flowRight( [
 			image: `/wp/v2/media/${ id }`,
 		};
 	} ),
+	connect(
+		( state ) => ( {
+			isSelectionEnabled: ! isSelectionDisable( state ),
+		} ),
+		{
+			toggleSelectionDisabled: toggleSelectionDisable,
+		}
+	),
```

Comment:
While this works, It raises the question of whether a block can subscribe to the state or not? We never didi this before and I think if we were to do it, it should be via the same mechanisms exposing the state for extensions. cc @aduth 

Do you think we can find a way to fix this bug in a generic way for all blocks?

Reply:
I updated this PR, now we don't access the store in the blocks. And we allow any block to disable/enable selection. I think when blocks implement a mouse movement equal to the one used for multi-selection they will need to disable multi-selection. I'm not seeing another possibility. The set of movements, in my opinion, is limited it will probably be resizing and drag and drop operations. And now we are proving a mechanism for blocks to solve this problem.